build/npm/refresh-langpack-extension.js

```diff
@@ -0,0 +1,187 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+let locFunc = require("../lib/locFunc");
+let fs = require("fs");
+let path = require("path");
+
+let gulp = require('gulp');
+let vfs = require("vinyl-fs");
+let rimraf = require('rimraf');
+let minimist = require('minimist');
+
+const textFields = {
+	"nameText": 'ads',
+	"displayNameText": 'Azure Data Studio',
+	"publisherText": 'Microsoft',
+	"licenseText": 'SEE SOURCE EULA LICENSE IN LICENSE.txt',
+	"updateText": 'cd ../vscode && npm run update-localization-extension '
+}
+
+//ADS language pack folder length
+const adsLangPackFolderLength = 17;
+
+//Extensions for ADS
+const currentADSJson = JSON.parse(fs.readFileSync(path.join(__dirname, '../../i18nExtensions/ADSExtensions.json'), 'utf8'));
+const currentADSExtensions = currentADSJson.ADSExtensions;
+const vscodeExtensions = currentADSJson.VSCODEExtensions;
+
+function runUpdateOnLanguages(){
+	let i18nPath =  path.join('.', 'i18n');
+	let i18nFolders = fs.readdirSync(i18nPath).filter(folderName => folderName.match(/ads-language-pack-[A-z]+/));
+	let langIds = i18nFolders.map(folderName => {return folderName.substring(adsLangPackFolderLength + 1);});
+	for(let langId in langIds){
+		update(langIds[langId]);
+	}
+}
+
+function update(langId) {
+	let idOrPath = langId;
+	if (!idOrPath) {
+		throw new Error('Argument must be the location of the localization extension.');
+	}
+	let location = path.join('.', 'resources', 'xlf');
+	if (location !== undefined && !fs.existsSync(location)) {
+		throw new Error(`${location} doesn't exist.`);
+	}
+	let locExtFolder = idOrPath;
+	if (/^\w{2}(-\w+)?$/.test(idOrPath)) {
+		locExtFolder = path.join('.', 'i18n', `ads-language-pack-${idOrPath}`);
+	}
+	let locExtStat = fs.statSync(locExtFolder);
+	if (!locExtStat || !locExtStat.isDirectory) {
+		throw new Error('No directory found at ' + idOrPath);
+	}
+	let packageJSON = JSON.parse(fs.readFileSync(path.join(locExtFolder, 'package.json')).toString());
+	//processing extension fields, version and folder name must be changed manually.
+	packageJSON['name'] = packageJSON['name'].replace('vscode', textFields.nameText);
```

Comment:
Why are we doing this here? This seems like it should just be fixed in the package.json instead (same with the other values)